test/integration/server.h

```diff
@@ -199,6 +199,10 @@ class IntegrationTestServer : Logger::Loggable<Logger::Id::testing>,
     return TestUtility::findCounter(*stat_store_, name);
   }
 
+  Stats::GaugeSharedPtr gauge(const std::string& name) {
+    return TestUtility::findGauge(*stat_store_, name);
```

Comment:
The same comment applies here as to counter() above. Can you add a similar comment or just have a comment that says something to the effect of "See counter() above for why this method exists."